client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -15,6 +15,8 @@ import { getStreamUrl } from 'reader/route';
 import EmailSettings from './email-settings';
 import { getSiteName, getSiteUrl } from 'reader/get-helpers';
 
+const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( /\/$/, '' );
```

Comment:
Can we add a comment explaining what this does to a URL, or make the name more descriptive?

& is it worth moving out to somewhere for reuse?